### PR TITLE
Update amphora image path for SOC9 to correct path

### PR DESCRIPTION
**Pull request by jeremy-moffitt**
The amphora image path in the Cloud 9 documentation refers to the wrong path. Currently the Cloud 9 docs refer to the Cloud 8 path, despite the image location and name changing in Cloud 9.

### CODE REVIEW

File: xml/installation-installation-configure_lbaas.xml [hunk 1]
```diff
@@ -161,7 +161,7 @@
    <step>
     <para>
      The full path and name for the Amphora image is
-     <filename>/srv/www/suse-12.3/x86_64/repos/Cloud/suse/noarch/openstack-octavia-amphora-image-x86_64-0.1.0-1.21.noarch.rpm</filename>
+     <filename>/srv/tftpboot/suse-12.4/x86_64/repos/Cloud/suse/noarch/openstack-octavia-amphora-image-x86_64-0.1.0-13.157.noarch.rpm</filename>
```
Comment (KeithMnemonic) [inline xml/installation-installation-configure_lbaas.xml]: Are you sure this is correct. On my patched SOC 9 it is in /srv/www/suse-12.4/x86_64/repos/Cloud/suse/noarch

Comment (KeithMnemonic) [reply xml/installation-installation-configure_lbaas.xml]: also is that version correct? This seems to imply a different version string .

https://build.suse.de/package/live_build_log/Devel:Cloud:9/openstack-octavia-amphora-image/SLE_12_SP4/x86_64

Comment (jeremy-moffitt) [reply xml/installation-installation-configure_lbaas.xml]: I got the path from @nicolasbock  , we should figure out why his environment has a different path than you're seeing

**Comment by asettle** [comment]
Also @jeremy-moffitt I'd also recommend pushing this into the master branch. I know we're not doing anymore active development, but let's not deviate. We should find a solution for the branch management here.

**Comment by jeremy-moffitt** [comment]
> Also @jeremy-moffitt I'd also recommend pushing this into the master branch. I know we're not doing anymore active development, but let's not deviate. We should find a solution for the branch management here.

Are we developing in master and cherry-picking to the maintenance branch or vice versa? I wasnt sure if master had deviated from the cloud 9 maintenance copy

File: xml/installation-installation-configure_lbaas.xml (hunk 1 above)
Comment (nicolasbock) [reply xml/installation-installation-configure_lbaas.xml]: On my fresh `develcloud9` env the path is
```
/srv/tftpboot/suse-12.4/x86_64/repos/Cloud/suse/noarch/openstack-octavia-amphora-image-x86_64-0.1.0-13.157.noarch.rpm
```

Comment (nicolasbock) [reply xml/installation-installation-configure_lbaas.xml]: I don't see much in `/srv/www`. Just two directories but no files.

Comment (jeremy-moffitt) [reply xml/installation-installation-configure_lbaas.xml]: is that CLM or Crowbar?

Comment (nicolasbock) [reply xml/installation-installation-configure_lbaas.xml]: Crowbar

Comment (jeremy-moffitt) [reply xml/installation-installation-configure_lbaas.xml]: My understanding from @KeithMnemonic is that the CLM path differs from the Crowbar path, will need him to confirm (or someone else with an Octavia CLM system setup)

**Comment by asettle** [comment]
> > Also @jeremy-moffitt I'd also recommend pushing this into the master branch. I know we're not doing anymore active development, but let's not deviate. We should find a solution for the branch management here.
> 
> Are we developing in master and cherry-picking to the maintenance branch or vice versa? I wasnt sure if master had deviated from the cloud 9 maintenance copy

We are indeed developing in master and cherry-picking. In theory, master shouldn't deviate from cloud 9 maintenance copy, but we need to manually backport each time.

**Comment by jeremy-moffitt** [comment]
updated screenshot:
amphora_up

**Comment by KeithMnemonic** [comment]
LGTM

**Pull request merged**